playwright.config.ts

```diff
@@ -4,39 +4,42 @@ import dotenv from "dotenv";
 dotenv.config();
 
 const env = process.env;
-const DEFAULT_RETRIES = "1";
 const DEFAULT_WORKERS = "2";
+// const DEFAULT_RETRIES = "1";
```

Comment:
do you need that comment?

Reply:
I want to see how disabling retries behaves and if needed revert this change.